protobuf/types/process.proto

```diff
@@ -138,38 +118,38 @@ message Process {
         ];
       }
 
-      // Key for the filter
-      string key = 1 [
-        (gogoproto.moretags) = 'hash:"name:1" validate:"required"'
-      ];
-
       // List of condition to apply for this filter
       repeated Condition conditions = 2 [
         (gogoproto.moretags) = 'hash:"name:2"',
         (gogoproto.nullable) = false
       ];
     }
 
+    // Key that identifies the node.
+    string key = 1 [
+      (gogoproto.moretags) = 'hash:"name:1" validate:"required"'
+    ];
+
     oneof type {
       // Result is a trigger that listens for a specific result.
-      Result result = 1 [
-        (gogoproto.moretags) = 'hash:"name:1"'
+      Result result = 2 [
```

Comment:
i think you don't need to start from 0 in a oneof

Reply:
from 0?

Reply:
i mean from 1